eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -374,9 +406,10 @@ function IsPythonPackageVersionPublished($pkgId, $pkgVersion) {
 # Retrieves the list of all tags that exist on the target repository
 function GetExistingTags($apiUrl) {
   try {
-    return (Invoke-WebRequest-WithHandling -Method "GET" -url "$apiUrl/git/refs/tags"  ) | % { $_.ref.Replace("refs/tags/", "") }
+    return (Invoke-WebRequest -Method "GET" -Uri "$apiUrl/git/refs/tags"  -MaximumRetryCount 3 -RetryIntervalSec 10) | % { $_.ref.Replace("refs/tags/", "") }
```

Comment:
What prompted this update?

Reply:
In this case `Invoke-WebRequest-WithHandling` will `exit(1)` if it "fails" instead of throwing an exception. After chatting w/ @scbedd we settled on using `Invoke-WebRequest` for this behavior because it'll throw when it encounters a 404. 

The API endpoint responds with a 404 error in the case that there are no tags in the repo. It doesn't respond with a 2xx and an empty list. 

This change reverts the behavior of this function to handle the originally written case where we fetch tags on a repo and there are no tags for that repo.